bindings/http/http_test.go

```diff
@@ -172,6 +172,69 @@ func TestInit(t *testing.T) {
 			path:      "/",
 			err:       "received status code 500",
 		},
+		"internal server error suppressed": {
```

Comment:
Not going to block the PR on this, but it would be great if you could add a test case for 2xx status codes too, such as 200 and 201 (for example).

Reply:
Sure, I will add some and push with the DCO issue solved -- can't hurt :D